docs/UserGuide.md

```diff
@@ -114,22 +90,82 @@ There are 5 CLI command formats as shown below:
+   `help 123` is interpreted as `help`.
+4) Command word is not case-sensitive: <br>
+   e.g. `list` and `LIST` are both valid commands.
 5) If you are using a PDF version of this document, be careful when copying and pasting commands that span multiple lines as space characters surrounding line-breaks may be omitted when copied over to the application.
+
+**Save and Storage information:**
+
+- AddressBook data are saved in the [hard disk](#glossary) automatically after any command that changes the data. There is no need to save manually.
+
+- AddressBook data are saved automatically as a JSON file `[JAR file location]/data/addressbook.json`. Advanced users are welcome to update data directly by editing that data file.
+
+<box type="warning" seamless>
+
+**Caution:**
+If your changes to the data file makes its format invalid, AddressBook will discard all data and start with an empty data file at the next run.  Hence, it is recommended to take a backup of the file before editing it.<br>
+Furthermore, certain edits can cause the AddressBook to behave in unexpected ways (e.g., if a value entered is outside the acceptable range). Therefore, edit the data file only if you are confident that you can update it correctly.
+</box>
+
+[Back to Table of Contents](#table-of-contents)
+
+--------------------------------------------------------------------------------------------------------------------
+
+# Input parameters
+
+<box type="info" seamless>
+
+**Notes about parameters:**<br>
+
+Parameters often take up the form of `p/[PARAMETER]` where p is the parameter symbol. For example:`add n/John Doe`<br>
+- `n/` -> parameter symbol<br>
+- `John Doe` -> parameter.
 </box>
 
-[Back to Input parameters](#input-parameters)<br>
+### Patient Parameters
+
+Symbol     | Parameter     | Constraints
+-----------|---------------|----------------------------------------------------------------------------------------------------------------------------------------------------------
+**`n`**| `NAME`        |- Must contain at least 1 alphabetic character and has a character limit of 50. <br> - Allows alphabetic characters, spaces, rounded brackets, hyphen, forward-slashes, @, and commas.
+**`i`**| `ID`          |- Must contain at least 1 alphanumeric character and has a character limit of 36. <br> - Allows alphanumeric characters, hyphens, forward-slashes, hashes, rounded brackets.
+**`w`**| `WARD`        |- Must contain at least 1 alphanumeric character and has a character limit of 50.
+**`d`**| `DIAGNOSIS`   |- Must contain at least 1 alphabetic character and has a character limit of 80.
+**`m`**| `MEDICATION`  |- Must contain at least 1 alphanumeric character and has a character limit of 80. <br> - Allows alphanumeric characters, spaces, commas, hyphen, forward-slashes, rounded brackets, periods.
+**`pn`**| `NOTES`       |- Allows any characters.
+**`a`**| `APPOINTMENT` |- Refer to [Appointment Parameters](#appointment-parameters) table below.
```

Comment:
do we have an "a/" format anywhere though?

Reply:
We have a `a/` format as the appointment description in `makeappt` command.
i.e. `makeappt 1 a/ Surgery s/ 01-01-2001-01-00 e/ 21-12-2024-23-59 `